##-------Question----------##
1.what is Output of following code?

class ThisClass{
            static void foo() throws Exception{
                           throw new Exception();
            }
            public static void main(String args[]){
                     try{
                              foo();
                         }catch(Exception e){
                                      System.exit(0);
                         }
                         finally{
                                   System.out.println("in finally");
                         }
           }
}
##ans1##
Nothing

##ans2##
print-In finally

##ans3##
Compiler Time error

##ans4##
RunTime Error

##ans5##

##correct##
1

##-------Question----------##
2.what is Output of following code?

class ThisClass{
            static void foo() throws Exception{
                           throw new Exception();
            }
            public static void main(String args[]){
                     try{
                              foo();
                         }catch(Exception e){
                                      System.exit(0);
                         }
                         finally{
                                   System.out.println("in finally");
                         }
           }
}
##ans1##
Nothing

##ans2##
print-In finally

##ans3##
Compiler Time error

##ans4##
RunTime Error

##ans5##

##correct##
1
##-------Question----------##
3.what is Output of following code?

class ThisClass{
            static void foo() throws Exception{
                           throw new Exception();
            }
            public static void main(String args[]){
                     try{
                              foo();
                         }catch(Exception e){
                                      System.exit(0);
                         }
                         finally{
                                   System.out.println("in finally");
                         }
           }
}
##ans1##
Nothing

##ans2##
print-In finally

##ans3##
Compiler Time error

##ans4##
RunTime Error

##ans5##

##correct##
1
##-------Question----------##
4.what is Output of following code?

class ThisClass{
            static void foo() throws Exception{
                           throw new Exception();
            }
            public static void main(String args[]){
                     try{
                              foo();
                         }catch(Exception e){
                                      System.exit(0);
                         }
                         finally{
                                   System.out.println("in finally");
                         }
           }
}
##ans1##
Nothing

##ans2##
print-In finally

##ans3##
Compiler Time error

##ans4##
RunTime Error

##ans5##

##correct##
1
##-------Question----------##
5.what is Output of following code?

class ThisClass{
            static void foo() throws Exception{
                           throw new Exception();
            }
            public static void main(String args[]){
                     try{
                              foo();
                         }catch(Exception e){
                                      System.exit(0);
                         }
                         finally{
                                   System.out.println("in finally");
                         }
           }
}
##ans1##
Nothing

##ans2##
print-In finally

##ans3##
Compiler Time error

##ans4##
RunTime Error

##ans5##

##correct##
1
##-------Question----------##
6.what is Output of following code?

class ThisClass{
            static void foo() throws Exception{
                           throw new Exception();
            }
            public static void main(String args[]){
                     try{
                              foo();
                         }catch(Exception e){
                                      System.exit(0);
                         }
                         finally{
                                   System.out.println("in finally");
                         }
           }
}
##ans1##
Nothing

##ans2##
print-In finally

##ans3##
Compiler Time error

##ans4##
RunTime Error

##ans5##

##correct##
1
##-------Question----------##
7.what is Output of following code?

class ThisClass{
            static void foo() throws Exception{
                           throw new Exception();
            }
            public static void main(String args[]){
                     try{
                              foo();
                         }catch(Exception e){
                                      System.exit(0);
                         }
                         finally{
                                   System.out.println("in finally");
                         }
           }
}
##ans1##
Nothing

##ans2##
print-In finally

##ans3##
Compiler Time error

##ans4##
RunTime Error

##ans5##

##correct##
1
##-------Question----------##
8.what is Output of following code?

class ThisClass{
            static void foo() throws Exception{
                           throw new Exception();
            }
            public static void main(String args[]){
                     try{
                              foo();
                         }catch(Exception e){
                                      System.exit(0);
                         }
                         finally{
                                   System.out.println("in finally");
                         }
           }
}
##ans1##
Nothing

##ans2##
print-In finally

##ans3##
Compiler Time error

##ans4##
RunTime Error

##ans5##

##correct##
1
##-------Question----------##
9.what is Output of following code?

class ThisClass{
            static void foo() throws Exception{
                           throw new Exception();
            }
            public static void main(String args[]){
                     try{
                              foo();
                         }catch(Exception e){
                                      System.exit(0);
                         }
                         finally{
                                   System.out.println("in finally");
                         }
           }
}
##ans1##
Nothing

##ans2##
print-In finally

##ans3##
Compiler Time error

##ans4##
RunTime Error

##ans5##

##correct##
1
##-------Question----------##
10.what is Output of following code?

class ThisClass{
            static void foo() throws Exception{
                           throw new Exception();
            }
            public static void main(String args[]){
                     try{
                              foo();
                         }catch(Exception e){
                                      System.exit(0);
                         }
                         finally{
                                   System.out.println("in finally");
                         }
           }
}
##ans1##
Nothing

##ans2##
print-In finally

##ans3##
Compiler Time error

##ans4##
RunTime Error

##ans5##

##correct##
1
##-------Question----------##
11.what is Output of following code?

class ThisClass{
            static void foo() throws Exception{
                           throw new Exception();
            }
            public static void main(String args[]){
                     try{
                              foo();
                         }catch(Exception e){
                                      System.exit(0);
                         }
                         finally{
                                   System.out.println("in finally");
                         }
           }
}
##ans1##
Nothing

##ans2##
print-In finally

##ans3##
Compiler Time error

##ans4##
RunTime Error

##ans5##

##correct##
1
##-------Question----------##
12.what is Output of following code?

class ThisClass{
            static void foo() throws Exception{
                           throw new Exception();
            }
            public static void main(String args[]){
                     try{
                              foo();
                         }catch(Exception e){
                                      System.exit(0);
                         }
                         finally{
                                   System.out.println("in finally");
                         }
           }
}
##ans1##
Nothing

##ans2##
print-In finally

##ans3##
Compiler Time error

##ans4##
RunTime Error

##ans5##

##correct##
1
##-------Question----------##
13.what is Output of following code?

class ThisClass{
            static void foo() throws Exception{
                           throw new Exception();
            }
            public static void main(String args[]){
                     try{
                              foo();
                         }catch(Exception e){
                                      System.exit(0);
                         }
                         finally{
                                   System.out.println("in finally");
                         }
           }
}
##ans1##
Nothing

##ans2##
print-In finally

##ans3##
Compiler Time error

##ans4##
RunTime Error

##ans5##

##correct##
1
##-------Question----------##
14.what is Output of following code?

class ThisClass{
            static void foo() throws Exception{
                           throw new Exception();
            }
            public static void main(String args[]){
                     try{
                              foo();
                         }catch(Exception e){
                                      System.exit(0);
                         }
                         finally{
                                   System.out.println("in finally");
                         }
           }
}
##ans1##
Nothing

##ans2##
print-In finally

##ans3##
Compiler Time error

##ans4##
RunTime Error

##ans5##

##correct##
1
##-------Question----------##
15.what is Output of following code?

class ThisClass{
            static void foo() throws Exception{
                           throw new Exception();
            }
            public static void main(String args[]){
                     try{
                              foo();
                         }catch(Exception e){
                                      System.exit(0);
                         }
                         finally{
                                   System.out.println("in finally");
                         }
           }
}
##ans1##
Nothing

##ans2##
print-In finally

##ans3##
Compiler Time error

##ans4##
RunTime Error

##ans5##

##correct##
1
##-------Question----------##
16.what is Output of following code?

class ThisClass{
            static void foo() throws Exception{
                           throw new Exception();
            }
            public static void main(String args[]){
                     try{
                              foo();
                         }catch(Exception e){
                                      System.exit(0);
                         }
                         finally{
                                   System.out.println("in finally");
                         }
           }
}
##ans1##
Nothing

##ans2##
print-In finally

##ans3##
Compiler Time error

##ans4##
RunTime Error

##ans5##

##correct##
1
##-------Question----------##
17.what is Output of following code?

class ThisClass{
            static void foo() throws Exception{
                           throw new Exception();
            }
            public static void main(String args[]){
                     try{
                              foo();
                         }catch(Exception e){
                                      System.exit(0);
                         }
                         finally{
                                   System.out.println("in finally");
                         }
           }
}
##ans1##
Nothing

##ans2##
print-In finally

##ans3##
Compiler Time error

##ans4##
RunTime Error

##ans5##

##correct##
1
##-------Question----------##
18.what is Output of following code?

class ThisClass{
            static void foo() throws Exception{
                           throw new Exception();
            }
            public static void main(String args[]){
                     try{
                              foo();
                         }catch(Exception e){
                                      System.exit(0);
                         }
                         finally{
                                   System.out.println("in finally");
                         }
           }
}
##ans1##
Nothing

##ans2##
print-In finally

##ans3##
Compiler Time error

##ans4##
RunTime Error

##ans5##

##correct##
1
##-------Question----------##
19.what is Output of following code?

class ThisClass{
            static void foo() throws Exception{
                           throw new Exception();
            }
            public static void main(String args[]){
                     try{
                              foo();
                         }catch(Exception e){
                                      System.exit(0);
                         }
                         finally{
                                   System.out.println("in finally");
                         }
           }
}
##ans1##
Nothing

##ans2##
print-In finally

##ans3##
Compiler Time error

##ans4##
RunTime Error

##ans5##

##correct##
1
##-------Question----------##
20.what is Output of following code?

class ThisClass{
            static void foo() throws Exception{
                           throw new Exception();
            }
            public static void main(String args[]){
                     try{
                              foo();
                         }catch(Exception e){
                                      System.exit(0);
                         }
                         finally{
                                   System.out.println("in finally");
                         }
           }
}
##ans1##
Nothing

##ans2##
print-In finally

##ans3##
Compiler Time error

##ans4##
RunTime Error

##ans5##

##correct##
1
##-------Question----------##
21.what is Output of following code?

class ThisClass{
            static void foo() throws Exception{
                           throw new Exception();
            }
            public static void main(String args[]){
                     try{
                              foo();
                         }catch(Exception e){
                                      System.exit(0);
                         }
                         finally{
                                   System.out.println("in finally");
                         }
           }
}
##ans1##
Nothing

##ans2##
print-In finally

##ans3##
Compiler Time error

##ans4##
RunTime Error

##ans5##

##correct##
1
##-------Question----------##
22.what is Output of following code?

class ThisClass{
            static void foo() throws Exception{
                           throw new Exception();
            }
            public static void main(String args[]){
                     try{
                              foo();
                         }catch(Exception e){
                                      System.exit(0);
                         }
                         finally{
                                   System.out.println("in finally");
                         }
           }
}
##ans1##
Nothing

##ans2##
print-In finally

##ans3##
Compiler Time error

##ans4##
RunTime Error

##ans5##

##correct##
1
##-------Question----------##
23.what is Output of following code?

class ThisClass{
            static void foo() throws Exception{
                           throw new Exception();
            }
            public static void main(String args[]){
                     try{
                              foo();
                         }catch(Exception e){
                                      System.exit(0);
                         }
                         finally{
                                   System.out.println("in finally");
                         }
           }
}
##ans1##
Nothing

##ans2##
print-In finally

##ans3##
Compiler Time error

##ans4##
RunTime Error

##ans5##

##correct##
1
##-------Question----------##
24.what is Output of following code?

class ThisClass{
            static void foo() throws Exception{
                           throw new Exception();
            }
            public static void main(String args[]){
                     try{
                              foo();
                         }catch(Exception e){
                                      System.exit(0);
                         }
                         finally{
                                   System.out.println("in finally");
                         }
           }
}
##ans1##
Nothing

##ans2##
print-In finally

##ans3##
Compiler Time error

##ans4##
RunTime Error

##ans5##

##correct##
1
##-------Question----------##
25.what is Output of following code?

class ThisClass{
            static void foo() throws Exception{
                           throw new Exception();
            }
            public static void main(String args[]){
                     try{
                              foo();
                         }catch(Exception e){
                                      System.exit(0);
                         }
                         finally{
                                   System.out.println("in finally");
                         }
           }
}
##ans1##
Nothing

##ans2##
print-In finally

##ans3##
Compiler Time error

##ans4##
RunTime Error

##ans5##

##correct##
1
##-------Question----------##
26.what is Output of following code?

class ThisClass{
            static void foo() throws Exception{
                           throw new Exception();
            }
            public static void main(String args[]){
                     try{
                              foo();
                         }catch(Exception e){
                                      System.exit(0);
                         }
                         finally{
                                   System.out.println("in finally");
                         }
           }
}
##ans1##
Nothing

##ans2##
print-In finally

##ans3##
Compiler Time error

##ans4##
RunTime Error

##ans5##

##correct##
1
##-------Question----------##
27.what is Output of following code?

class ThisClass{
            static void foo() throws Exception{
                           throw new Exception();
            }
            public static void main(String args[]){
                     try{
                              foo();
                         }catch(Exception e){
                                      System.exit(0);
                         }
                         finally{
                                   System.out.println("in finally");
                         }
           }
}
##ans1##
Nothing

##ans2##
print-In finally

##ans3##
Compiler Time error

##ans4##
RunTime Error

##ans5##

##correct##
1
##-------Question----------##
28.what is Output of following code?

class ThisClass{
            static void foo() throws Exception{
                           throw new Exception();
            }
            public static void main(String args[]){
                     try{
                              foo();
                         }catch(Exception e){
                                      System.exit(0);
                         }
                         finally{
                                   System.out.println("in finally");
                         }
           }
}
##ans1##
Nothing

##ans2##
print-In finally

##ans3##
Compiler Time error

##ans4##
RunTime Error

##ans5##

##correct##
1
##-------Question----------##
29.what is Output of following code?

class ThisClass{
            static void foo() throws Exception{
                           throw new Exception();
            }
            public static void main(String args[]){
                     try{
                              foo();
                         }catch(Exception e){
                                      System.exit(0);
                         }
                         finally{
                                   System.out.println("in finally");
                         }
           }
}
##ans1##
Nothing

##ans2##
print-In finally

##ans3##
Compiler Time error

##ans4##
RunTime Error

##ans5##

##correct##
1
##-------Question----------##
30.what is Output of following code?

class ThisClass{
            static void foo() throws Exception{
                           throw new Exception();
            }
            public static void main(String args[]){
                     try{
                              foo();
                         }catch(Exception e){
                                      System.exit(0);
                         }
                         finally{
                                   System.out.println("in finally");
                         }
           }
}
##ans1##
Nothing

##ans2##
print-In finally

##ans3##
Compiler Time error

##ans4##
RunTime Error

##ans5##

##correct##
1
##-------Question----------##
31.what is Output of following code?

class ThisClass{
            static void foo() throws Exception{
                           throw new Exception();
            }
            public static void main(String args[]){
                     try{
                              foo();
                         }catch(Exception e){
                                      System.exit(0);
                         }
                         finally{
                                   System.out.println("in finally");
                         }
           }
}
##ans1##
Nothing

##ans2##
print-In finally

##ans3##
Compiler Time error

##ans4##
RunTime Error

##ans5##

##correct##
1
##-------Question----------##
32.what is Output of following code?

class ThisClass{
            static void foo() throws Exception{
                           throw new Exception();
            }
            public static void main(String args[]){
                     try{
                              foo();
                         }catch(Exception e){
                                      System.exit(0);
                         }
                         finally{
                                   System.out.println("in finally");
                         }
           }
}
##ans1##
Nothing

##ans2##
print-In finally

##ans3##
Compiler Time error

##ans4##
RunTime Error

##ans5##

##correct##
1
##-------Question----------##
33.what is Output of following code?

class ThisClass{
            static void foo() throws Exception{
                           throw new Exception();
            }
            public static void main(String args[]){
                     try{
                              foo();
                         }catch(Exception e){
                                      System.exit(0);
                         }
                         finally{
                                   System.out.println("in finally");
                         }
           }
}
##ans1##
Nothing

##ans2##
print-In finally

##ans3##
Compiler Time error

##ans4##
RunTime Error

##ans5##

##correct##
1
##-------Question----------##
34.what is Output of following code?

class ThisClass{
            static void foo() throws Exception{
                           throw new Exception();
            }
            public static void main(String args[]){
                     try{
                              foo();
                         }catch(Exception e){
                                      System.exit(0);
                         }
                         finally{
                                   System.out.println("in finally");
                         }
           }
}
##ans1##
Nothing

##ans2##
print-In finally

##ans3##
Compiler Time error

##ans4##
RunTime Error

##ans5##

##correct##
1
##-------Question----------##
35.what is Output of following code?

class ThisClass{
            static void foo() throws Exception{
                           throw new Exception();
            }
            public static void main(String args[]){
                     try{
                              foo();
                         }catch(Exception e){
                                      System.exit(0);
                         }
                         finally{
                                   System.out.println("in finally");
                         }
           }
}
##ans1##
Nothing

##ans2##
print-In finally

##ans3##
Compiler Time error

##ans4##
RunTime Error

##ans5##

##correct##
1
##-------Question----------##
36.what is Output of following code?

class ThisClass{
            static void foo() throws Exception{
                           throw new Exception();
            }
            public static void main(String args[]){
                     try{
                              foo();
                         }catch(Exception e){
                                      System.exit(0);
                         }
                         finally{
                                   System.out.println("in finally");
                         }
           }
}
##ans1##
Nothing

##ans2##
print-In finally

##ans3##
Compiler Time error

##ans4##
RunTime Error

##ans5##

##correct##
1

##-------Question----------##
37.what is Output of following code?

class ThisClass{
            static void foo() throws Exception{
                           throw new Exception();
            }
            public static void main(String args[]){
                     try{
                              foo();
                         }catch(Exception e){
                                      System.exit(0);
                         }
                         finally{
                                   System.out.println("in finally");
                         }
           }
}
##ans1##
Nothing

##ans2##
print-In finally

##ans3##
Compiler Time error

##ans4##
RunTime Error

##ans5##

##correct##
1
##-------Question----------##
38.what is Output of following code?

class ThisClass{
            static void foo() throws Exception{
                           throw new Exception();
            }
            public static void main(String args[]){
                     try{
                              foo();
                         }catch(Exception e){
                                      System.exit(0);
                         }
                         finally{
                                   System.out.println("in finally");
                         }
           }
}
##ans1##
Nothing

##ans2##
print-In finally

##ans3##
Compiler Time error

##ans4##
RunTime Error

##ans5##

##correct##
1
##-------Question----------##
39.what is Output of following code?

class ThisClass{
            static void foo() throws Exception{
                           throw new Exception();
            }
            public static void main(String args[]){
                     try{
                              foo();
                         }catch(Exception e){
                                      System.exit(0);
                         }
                         finally{
                                   System.out.println("in finally");
                         }
           }
}
##ans1##
Nothing

##ans2##
print-In finally

##ans3##
Compiler Time error

##ans4##
RunTime Error

##ans5##

##correct##
1
##-------Question----------##
what is Output of following code?

class ThisClass{
            static void foo() throws Exception{
                           throw new Exception();
            }
            public static void main(String args[]){
                     try{
                              foo();
                         }catch(Exception e){
                                      System.exit(0);
                         }
                         finally{
                                   System.out.println("in finally");
                         }
           }
}
##ans1##
Nothing

##ans2##
print-In finally

##ans3##
Compiler Time error

##ans4##
RunTime Error

##ans5##

##correct##
1
